mlflow/utils/cli_args.py

```diff
@@ -45,12 +49,49 @@
 NO_CONDA = click.option(
     "--no-conda",
     is_flag=True,
-    help="If specified, will assume that MLmodel/MLproject is running within "
+    help="[Deprecated] If specified, will assume that MLmodel/MLproject is running within "
     "a Conda environment with the necessary dependencies for "
     "the current project instead of attempting to create a new "
     "conda environment.",
 )
 
+
+ENV_MANAGER = click.option(
+    "--env-manager",
+    default=None,
+    required=False,
+    type=click.UNPROCESSED,
+    callback=lambda ctx, param, value: None if value is None else EnvManager.from_string(value),
+    help="If specified, create an environment for MLmodel/MLproject using the specified "
+    "environment manager. Valid values are ['local', 'conda']. If unspecified, default to "
+    "'conda'.",
```

Comment:
Can we explain what each of this values means? I.e. local -> use the local environment. conda -> use conda.

Reply:
Sure!